doc/src/build/devnet.md

```diff
@@ -46,19 +46,18 @@ To connect the Sui client to Sui Devnet, run the following command:
 ```shell
 $ sui client
 ```
-The Sui CLI prints the following line when the client starts up for the first time:
-```shell
-Config file ["/Users/dir/.sui/sui_config/client.yaml"] doesn't exist, do you want to connect to a Sui RPC server [y/n]?
-```
-Type `y` and then press `Enter`. You should see the following output:
+
+The Sui client defaults to use localnet environment. You can also change it to Devnet fullnode RPC.
+
+Previously installed a Sui client with a local network, modify your existing  `client.yaml` to change the RPC endpoint to connect to Sui Devnet fullnode with the following commands:
 ```shell
-Sui RPC server Url (Default to Sui Devnet if not specified) :
+$ sui client switch --env devnet
 ```
-The Sui client prompts you for the RPC server URL; press 'Enter' and it defaults to Devnet. You can also enter a custom URL to connect to a custom RPC endpoint.
 
-If you previously installed a Sui client with a local network, modify your existing  `client.yaml` to change the RPC endpoint with the following command:
+To add a custom URL to connect to a custom RPC endpoint:
 ```shell
-$ sui client switch --rpc https://fullnode.devnet.sui.io:443
+$ sui client new-env --alias <ALIAS> --rpc <RPC>
```

Comment:
I was going to make this RPC URL since the value for --rpc is a URL vs the string for the alias name.
```suggestion
$ sui client new-env --alias AliasName --rpc https://node.url:443
```
I've requested an update to the --help for this to clarify as well.